cherrypy/test/sessiondemo.py

```diff
@@ -123,9 +123,12 @@ def page(self):
             'reqcookie': cherrypy.request.cookie.output(),
             'sessiondata': list(cherrypy.session.items()),
             'servertime': (
-                datetime.utcnow().strftime('%Y/%m/%d %H:%M') + ' UTC'
+                datetime.now(UTC).strftime('%Y/%m/%d %H:%M UTC')
+            ),
+            'serverunixtime':
+            calendar.timegm(
+                datetime.datetime.utcnow(UTC).timetuple()
```

Comment:
Looks like this one place has the nested datetime forgotten:
```suggestion
                datetime.utcnow(UTC).timetuple(),
```